mod_prometheus.rs

```diff
@@ -316,36 +316,61 @@ fn prometheus_load(mod_int: &ModInterface) -> Status {
 
     // Channel hangup complete
     id = freeswitchrs::event_bind("mod_prometheus", fsr::event_types::CHANNEL_HANGUP_COMPLETE, None, |e| {
-        if let Some(last_bridge_hup_cause) = e.header("variable_last_bridge_hangup_cause") {
-            if last_bridge_hup_cause == "NORMAL_CLEARING" {
+
+        if let Some(callerIdName) = e.header("Caller-Caller-ID-Name"){
+            if let Some(uniqueId )= e.header("Unique-ID") {
```

Comment:
better to use the variable_sip_call_id (or maybe extract the 2?)  to be able to relate to actual calls and traces.